docs/api/robottelo.cli.rst

```diff
@@ -35,6 +35,16 @@ Submodules:
 
 .. automodule:: robottelo.cli.contentview
 
+:mod:`robottelo.cli.discoveryhost`
```

Comment:
my suggestion is to use `cli.discoveredhost` because that's naming convention we are following for UI